.idea/markdown-navigator-enh.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These `.idea/` files can be ignored - I had updated my machine's version of Android Studio